apps/docs/pages/guides/realtime/extensions/postgres-changes.mdx

```diff
@@ -352,10 +352,10 @@ const { createClient } = require('@supabase/supabase-js')
 const supabase = createClient(process.env.SUPABASE_URL, process.env.SUPABASE_KEY, {
   realtime: {
     headers: {
-      apikey: 'supabase-signed-token',
+      apikey: `Bearer ${your_custom_token}`,
```

Comment:
Confirming this should be `Bearer` for `apikey` (vs. `authorization`)?

Reply:
Yeah it's odd, but it's what seems to work. I'd like to figure out exactly why.

Reply:
Interesting - no `Bearer` prefix for non-realtime endpoints - agreed would be good to understand this